packages/cli/subtasks/preview-call.js

```diff
@@ -1,25 +1,14 @@
 const { subtask } = require('hardhat/config');
 const { SUBTASK_PREVIEW_CALL } = require('../task-names');
 const logger = require('@synthetixio/core-js/utils/logger');
-const prompter = require('@synthetixio/core-js/utils/prompter');
```

Comment:
Bunch of functionality moved to the `execute-call` subtask because it makes more sense there.

This subtask will be kinda skinny for now, but I think we will add stuff to it like gas price, gas limit, sender balance, etc.